content/en/technical-overview/architecture/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "CHT Architecture"
+linkTitle: "Architecture"
+weight: 1
+aliases:
+   - /core
+   - /core/overview/
+---
+
+### Components of the CHT Ecosystem
```

Comment:
```suggestion
```

Reply:
I suggest we move this header to `description` so we can see it in the `relatedContent` pages. 

See the current reference without the `description`:


<img width="570" alt="Screenshot 2025-05-27 at 10 37 26" src="https://github.com/user-attachments/assets/a1e191b8-8466-4483-b332-6068ee78988d" />